portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -59,20 +60,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) {
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) {
         //Run a query to see if we get any results from the video ID of the POST request
-        String id = request.getParameter(PropertyNames.VIDEO_ID);
-        if (id == null || id.isEmpty()) {
+        String videoId = request.getParameter(PropertyNames.VIDEO_ID);
+        if (videoId == null || videoId.isEmpty()) {
             sendErrorMessage(response, HttpServletResponse.SC_BAD_REQUEST, "Video ID cannot be empty.");
             return;
         }
         Query query = new Query("Video")
-            .setFilter(new FilterPredicate(PropertyNames.VIDEO_ID, FilterOperator.EQUAL, id));
+            .setFilter(new FilterPredicate(PropertyNames.VIDEO_ID, FilterOperator.EQUAL, videoId));
         PreparedQuery preparedQuery = datastore.prepare(query);
         Entity queryVideoEntity = preparedQuery.asSingleEntity();
         //if the video Id doesnt exist in our database, we convert the request to a video entity, add it to the database, and redirect. 
```

Comment:
NIT: I know this might not have been part of your PR, but please fix grammar.

Reply:
Fixed.